momento-sdk/src/main/java/momento/sdk/TokenGrpcStubsManager.java

```diff
@@ -0,0 +1,64 @@
+package momento.sdk;
+
+import io.grpc.ClientInterceptor;
+import io.grpc.ManagedChannel;
+import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder;
+import java.io.Closeable;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nonnull;
+import momento.sdk.auth.CredentialProvider;
+import momento.token.TokenGrpc;
+
+/**
+ * Manager responsible for GRPC channels and stubs for the Disposable Token.
+ *
+ * <p>The business layer, will get request stubs from this layer. This keeps the two layers
+ * independent and any future pooling of channels can happen exclusively in the manager without
+ * impacting the API business logic.
+ */
+final class TokenGrpcStubsManager implements Closeable {
```

Comment:
for consistency with the other internal stubs managers we might want to prefix this with `Scs`.

Alternately since the name `Scs` is kind of outdated we could change the prefix for all of the internal classes to be `Internal` or something.